Automattic/wp-calypso
# Refactor user settings selectors to TS and remove lodash

#### Changes proposed in this Pull Request

* This PR refactors the `getUnsavedUserSettings()`, `getUserSetting()`, and `getUserSettings()` selectors to be written in TypeScript, and removes their use of the `get()` function from `lodash`

#### Testing instructions

* It will be pretty difficult to test this directly, but it may be easier to make sure that `/me/account` continues to work as expected when updating various values.

Related to #61734 , which added the types at a higher level (and not in the right place)
